src/Arcus.Templates.Tests.Integration/Arcus.Templates.Tests.Integration.csproj

```diff
@@ -24,12 +24,19 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="Arcus.EventGrid.Testing" Version="2.3.0" />
+    <PackageReference Include="Arcus.WebApi.Security.Authentication" Version="0.2" />
     <PackageReference Include="Flurl.Http" Version="2.4.2" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
     <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Polly" Version="7.1.1" />
     <PackageReference Include="xunit" Version="2.4.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <Compile Update="Fixture\InMemorySecretProvider.cs">
```

Comment:
Is this on purpose?

Reply:
Yep, that way we can manipulate the to be created project from the template. Can change it a little bit so we don't have this though.

Reply:
Changed. Not anymore in the project file.